src/sentry/static/sentry/app/components/loading/loadingContainer.tsx

```diff
@@ -22,22 +37,19 @@ export default function LoadingContainer(props) {
   );
 }
 
+LoadingContainer.defaultProps = defaultProps;
+
 LoadingContainer.propTypes = {
   isLoading: PropTypes.bool.isRequired,
   isReloading: PropTypes.bool.isRequired,
   children: PropTypes.node,
 };
 
-LoadingContainer.defaultProps = {
-  isLoading: false,
-  isReloading: false,
-};
-
 const Container = styled('div')`
   position: relative;
 `;
 
-const LoadingMask = styled('div')`
+const LoadingMask = styled('div')<MaskProps>`
```

Comment:
IMO just inline the type here

```jsx
const LoadingMask = styled('div')<{isReloading: boolean}>`

```

Reply:
I find inline types can get unwieldy and hard to read pretty quickly. In this situation it could work but generally I find separate types easier to grok.